### DISCUSSION TITLE
Does anyone know what is this symbol is

### DISCUSSION THREAD

**Discussion by 0xt3d** [starship/starship#3971] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/34211458/167234397-f5a22207-04b0-4d6c-93a9-9abda222d7bb.png)

✦2 <- this one

**Comment by syphar** [comment, marked as answer]
calling `starship explain` will give you the explanation. 

I know the `✦` symbol from the `jobs` module in starship which would mean there are suspended / background jobs running

**Reply by 0xt3d** [reply]
Thank you for the explanation

**Comment by andytom** [comment]
That looks like the [jobs module](https://starship.rs/config/#jobs).

**Comment by albheim** [comment]
Started getting this from any command that change directory. And I can't analyze it with `starship explain` since it seems to not catch it.

I find it slightly annoying in this case, though I quite like the idea otherwise since I do occasionally pause jobs on purpose and then the visual indication is good. What would be a good way to get rid of it for the cd case, but still keep the functionality?

```
~
❯ cd Downloads         # After cd we have the glyph

~/Downloads
✦ ❯ j other            # After autojump we also have glyph
.../otherdir

~/otherdir
✦ ❯ ls                 # After ls no glyph

~/otherdir
❯ cd

~
✦ ❯                    # After empty command no glyph

~
❯ cd

~
✦ ❯ starship explain   # Seems to not catch it here

 Here's a breakdown of your prompt:
 "~ " (<1ms)  -  The current working directory
 "❯ " (<1ms)  -  A character (usually an arrow) beside where the text is entered in your terminal
```

**Reply by chipbuster** [reply]
Autojump is known to cause this issue, especially with shells like bash where there's no way to distinguish between general background jobs and background jobs which are triggered for prompt draw (though I think it also shows up to some extent in other shells with autojump).

As far as I know, the only solution right now is to use something like [z.sh](https://github.com/rupa/z) or [zoxide](https://github.com/ajeetdsouza/zoxide).

**Reply by albheim** [reply]
Okay, zoxide looks nice so might give that a try instead of autojump then. Thanks for the tip!

**Reply by llagerlof** [reply]
Just to add my experience with autojump and starship, this bug isn't happening for me. Autojump is not causing this issue **in my case**.

Autojump version: 22.5.3

starship version:  1.2.1

Bash version: 5.1.16(1)-release
